packages/core/src/components/DataTable/columns/ExpandableColumn.tsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import { Column } from 'react-virtualized';
+import IconChevronDown from '@airbnb/lunar-icons/lib/interface/IconChevronDown';
+import IconChevronRight from '@airbnb/lunar-icons/lib/interface/IconChevronRight';
+import { css, WithStylesProps } from '../../../composers/withStyles';
+import Spacing from '../../Spacing';
+import { TableRow } from '../types';
+import { EXPANDABLE_COLUMN_WIDTH } from '../constants';
+
+export default function ExpandableColumn(
+  styles: WithStylesProps['styles'],
+  expandedRows: Set<number>,
+  expandRow: (newExpandedRowIndex: number) => (event: any) => void,
+) {
+  const cellRenderer = (row: TableRow) => {
+    const { children, originalIndex } = row.rowData.metadata;
+
+    if (children && children.length > 0) {
+      const Chevron = expandedRows.has(originalIndex) ? IconChevronDown : IconChevronRight;
+
+      return (
+        <div
+          role="button"
+          tabIndex={0}
+          {...css(styles.chevron)}
+          onClick={expandRow(originalIndex)}
+          onKeyPress={expandRow(originalIndex)}
+        >
+          <Spacing left={1.5}>
```

Comment:
Does this need `inline`?

Reply:
I don't think so, it looks right to me

Reply:
<img width="293" alt="Screen Shot 2019-04-22 at 2 59 17 PM" src="https://user-images.githubusercontent.com/8676510/56534798-38995b00-650f-11e9-863a-b2c4aea6b633.png">